--- bogoseek-0.3.5/Makefile~	2008-11-23 10:57:28.000000000 +0100
+++ bogoseek-0.3.5/Makefile	2008-12-28 13:50:20.159574738 +0100
@@ -1,6 +1,8 @@
 VERSION=0.3.5
+CC=gcc
 CFLAGS=-O2 -Wall
-LDFLAGS=-lm -lrt
+LDFLAGS=
+LIBS=-lm -lrt
 
 prefix = /usr
 exec_prefix = ${prefix}
@@ -8,6 +10,13 @@
 
 all: hist bogoseek
 
+hist: hist.c
+	$(CC) $(CFLAGS) $(LDFLAGS) $< -o $@ $(LIBS)
+
+bogoseek: bogoseek.c
+	$(CC) $(CFLAGS) $(LDFLAGS) $< -o $@ $(LIBS)
+
+
 distclean: clean
 
 clean:;
